src/Build/BackEnd/Components/Logging/LoggingService.cs

```diff
@@ -1594,6 +1595,8 @@ private void InitializeLogger(ILogger logger, IEventSource sourceForLogger)
         {
             try
             {
+                // TODO: can we distinguish builtin and custom loggers here?
```

Comment:
Builtin are probably defined in assemblies that are anyway already loaded, so we might not care...
Any thoughts here are appreciated.

Same applies for SdkResolvers

Reply:
Feels like we _could_ have an explicit check for the known types, but there are enough of them that I don't know how helpful it would be.